### ISSUE TITLE
why so strict on the casacore version?

### ISSUE THREAD

**Issue by o-smirnov** [ratt-ru/CubiCal#328]
https://github.com/ratt-ru/CubiCal/blob/master/setup.py#L56 specifies <=3.0.0. Any particular reason @gijzelaerr @JSKenyon @bennahugo?

I'm pretty confused about the PyPI status of casacore, and I keep banging my shins on it.

**Comment by o-smirnov** [comment]
While we're on the subject, @bennahugo could you please remind me what was the fix for cubical bailing out with
```
SEVERE  MeasTable::dUTC(Double) (file /build/casacore-3.1.
1/measures/Measures/MeasTable.cc, line 4281)    Cannot read leap second table TAI_UTC
```
You told me last week, but my brain has dumped core. It was something about the current wheel being broken and needing to build from source? It's all Afrikaans to me!

**Comment by bennahugo** [comment]
I've had very little luck with the pypi wheels - neither the 3.0 nor the 3.1 wheels work correctly. I've just run into library import problems with the 3.1 wheel when I ran meqtrees using python 3. 

I've fixed it in cubical to 3.0 because 3.1 didn't compile against the 16.04 LTS kern distributed casacore when I set this up. I can move the release to 3.1, but it will mean we need to start compiling casacore from source on all 16.04 systems. 

To fix your current issue you need to use the ```--no-binary``` switch

**Comment by bennahugo** [comment]
This leads to something that we should think about when @gijzelaerr comes down to work on Kern. Is it possible to automate PPA uploads on successful builds. If so can we move to a rolling release system where all supported LTS versions receive periodic updates instead of the current single release per LTS? I find that we inevitably need to start compiling all our software from source midway through the lifespan of LTS versions to keep production going. This happened with KERN 3 on 14.04 and now again with 16.04.
